content/en/docs/set-up/configuration.md

```diff
@@ -234,6 +234,358 @@ Variables specific to the _Lotus daemon_:
 - `LOTUS_CHAIN_INDEX_CACHE`: Sets the size for the epoch index cache. Defaults to `32768`. Increase if you perform frequent deep chain lookups for block heights far from the latest height.
 - `LOTUS_BSYNC_MSG_WINDOW`: Sets the initial maximum window size for message fetching blocksync request. Set to 10-20 if you have an internet connection with low bandwidth.
 
+Variables for resource allocation tuning (overrides settings in the resource allocation table).
+
+* 32G environment:
+```
+AP_32G_BASE_MIN_MEMORY=1073741824
+AP_32G_GPU_UTILIZATION=0
+AP_32G_MAX_MEMORY=4294967296
+AP_32G_MAX_PARALLELISM=1
+AP_32G_MAX_PARALLELISM_GPU=0
+AP_32G_MIN_MEMORY=4294967296
+C1_32G_BASE_MIN_MEMORY=1073741824
```

Comment:
These things are super long. I'm gonna try and add them into a `<details>` tag.

Reply:
I wasnt sure of the best way to format it, but a drop down is a great idea.

Reply:
Adds a `<details>` shortcode: https://github.com/filecoin-project/lotus-docs/pull/61